commerce_coordinator/apps/stripe/clients.py

```diff
@@ -0,0 +1,101 @@
+"""
+API clients for Stripe.
+"""
+import stripe
+from celery.utils.log import get_task_logger
+from django.conf import settings
+
+# Use special Celery logger for tasks client calls.
+logger = get_task_logger(__name__)
+
+
+class StripeAPIClient:
+    """
+    API client for calls to Stripe using API key.
+    """
+
+    def __init__(self):
+        configuration = settings.PAYMENT_PROCESSOR_CONFIG['edx']['stripe']
+
+        # The secret API key used by the backend to communicate with Stripe. Private/secret.
+        stripe.api_key = configuration['secret_key']
+        # Stripe API version to use. Will use latest allowed in Stripe Dashboard if None.
+        stripe.api_version = configuration['api_version']
+        # Send anonymous latency metrics to Stripe.
+        stripe.enable_telemetry = configuration['enable_telemetry']
+        # Stripe client logging level. None will default to INFO.
+        stripe.log = configuration['log_level']
+        # How many times to automatically retry requests. None means no retries.
+        stripe.max_network_retries = configuration['max_network_retries']
+        # Send requests somewhere else instead of Stripe. May be useful for testing.
+        stripe.proxy = configuration['proxy']
```

Comment:
Good point. The settings already in base.py will not always be what we want to use for our local/dev environment. Your comment also made me realize that the `log_level` in settings/base.py is incorrect.

I added `PAYMENT_PROCESSOR_CONFIG` to settings/local.py and fixed the base.py `log_level` in 3ea8992b7bff226764ab49e889d0964985c2db2c.

Musing here, but we'll probably have to set `PAYMENT_PROCESSOR_CONFIG` manually in settings/private.py in the future too, because we shouldn't commit our secrets (even local dev secrets for our Stripe account) publicly.